app/controllers/render-tree.js

```diff
@@ -12,6 +13,44 @@ export default Controller.extend({
   modelEmpty: equal('model.length', 0),
   showEmpty: and('initialEmpty', 'modelEmpty'),
 
+  /**
+   * Service used for storage. Storage is
+   * needed for remembering if the user closed the warning
+   * as it might get mildly annoying for devs to see and close
+   * the trival warning everytime.
+   * The default storage service is local storage however we
+   * fall back to memory storage if local storage is disabled (For
+   * example as a security setting in Chrome).
+   *
+   * @property storage
+   * @return {Service}
+   */
+  storage: service(`storage/${LocalStorageService.SUPPORTED ? 'local' : 'memory'}`),
+
+  /**
+   * Checks if the user previously closed the warning by referencing localStorage
+   * it is a computed get/set property.
+   *
+   * @property isClosed
+   * @type {Boolean}
+   */
+  isClosed: computed({
+    get() {
+      return !!this.get('storage').getItem('isClosed');
```

Comment:
Please rename this to `is-render-tree-warning-closed`. Storage is shared across the entire app so we need to avoid naming conflicts.